docs/docs/30-administration/70-proxy.md

```diff
@@ -85,7 +85,13 @@ This guide provides a brief overview for installing Woodpecker server behind the
 woodpecker.example.com {
   reverse_proxy woodpecker-server:8000
 }
+
+woodpeckeragent.example.com {
+  reverse_proxy h2c://woodpecker-server:9000
+}
 ```
+Above configuration proxies both ports for web and agent comunication. If your agent uses ssl do not forget to enable WOODPECKER_GRPC_SECURE. 
```

Comment:
```suggestion
```

:::note
Above configuration shows how to create reverse-proxies for web and agent communication. If your agent uses SSL do not forget to enable WOODPECKER_GRPC_SECURE.
:::
```